filesystems/sra/src/main/scala/cromwell/filesystems/sra/SraPathBuilderFactory.scala

```diff
@@ -0,0 +1,15 @@
+package cromwell.filesystems.sra
+
+import akka.actor.ActorSystem
+import com.typesafe.config.Config
+import cromwell.core.WorkflowOptions
+import cromwell.core.path.PathBuilderFactory
+
+import scala.concurrent.{ExecutionContext, Future}
+
+final case class SraPathBuilderFactory(globalConfig: Config, instanceConfig: Config)
+  extends PathBuilderFactory {
+  def withOptions(options: WorkflowOptions)(implicit as: ActorSystem, ec: ExecutionContext): Future[SraPathBuilder] = Future {
+    new SraPathBuilder()
+  }
```

Comment:
This could be a `Future.successful()` instead since there's no real reason to do it asynchronously
Also if the `SraPathBuilder` does not depend on workflow option we could even have a single instance somewhere in an `object`

Reply:
Good point.  Done.